hack/make.sh

```diff
@@ -115,7 +115,7 @@ if [ "$AUTO_GOPATH" ]; then
 	if [ "$(go env GOOS)" = 'solaris' ]; then
 		# sys/unix is installed outside the standard library on solaris
 		# TODO need to allow for version change, need to get version from go
-		export GO_VERSION=${GO_VERSION:-"1.7"}
+		export GO_VERSION=${GO_VERSION:-"1.7.1"}
 		export GOPATH="${GOPATH}:/usr/lib/gocode/${GO_VERSION}"
```

Comment:
Looks like that value used to be hardcoded. I'm interpreting the comment as meaning they want a better way to automatically get the latest go version so every time it's updated people don't have to manually do it here. In that case we don't even need GO_VERSION here because it's already there from the Dockerfile (it is just checked). You agree?